src/commands/force/user/create.ts

```diff
@@ -71,6 +71,10 @@ export class UserCreateCommand extends SfdxCommand {
       char: 'f',
       description: messages.getMessage('flags.definitionfile'),
     }),
+    setpassedusernameunique: flags.boolean({
+      char: 's',
```

Comment:
We use -s for "--setdefaultusername" a lot, so I don't think we should use it here for a different action, might be confusing.  As I said above, I like "-q", but I also defer to what others think.

Reply:
@jshackell-sfdc I used `-s`. As you said, `-u` can't be used because it's a generic shortcut for all commands, however I think that `-s` isn't used on this one.

I've already see several commands using the same shortcut for several options, for instance `-p` is used in `force:source:retrieve` as a shortcut for `--sourcepath` whereas for `force:package:install` it's a shortcut of `--package`.

So I don't think it's an issue to use it, but it's up to you.

Reply:
Actually I just saw that `-s` is already used for another shortcut than `--setdefaultusername`, for instance in `force:package:install` it's a shortcut for `--securitytype`.

Reply:
[We officially say that -s is used for sObject values,](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_plugins.meta/sfdx_cli_plugins/cli_plugins_style_parameter_names.htm) but we "break" that rule all the time, as you can see with the other uses of -s.  New param names are up to the group, so if we all think -s is fine, then I'm fine with it too!  I'm just the tech writer :)

Reply:
Ok, I'll keep `-s` then as it's the easier to guess, it's already used for other shortcuts on some other commands, and looking at what this command is doing they'll never be a `--sobjecttype` flag on this one.